mysql-test/suite/binlog/t/binlog_stm_delete_with_compress_column.test

```diff
@@ -0,0 +1,5 @@
+# This is a wrapper for binlog_delete_with_compress_column.test so that the same
+# test case can be used for both statement and row based logging
+
+-- source include/have_binlog_format_mixed_or_statement.inc
+-- source extra/binlog_tests/binlog_delete_with_compress_column.test
```

Comment:
I don't think there is no any need to run the test in statement format, as corrupted data is not propagated in statement based replication. So, I suggest you to please remove this test.